packages/relayer/docker-compose/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     container_name: prometheus
     restart: always
     volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
is this change intended for the relayer, and not a new folder for balance-monitor?

Reply:
Updated.